dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/GitHubSourceCodeRepo.java

```diff
@@ -1063,6 +1063,7 @@ public void syncUserMetadataFromGitHubByUsername(User user, TokenDAO tokenDAO) {
     }
 
     public User.Profile getProfile(final User user, final GHUser ghUser) throws IOException {
+        LOG.info("GitHub user profile id is " + ghUser.getId() + " and GitHub username is " + ghUser.getLogin() + " for Dockstore user " + user.getUsername());
```

Comment:
Code smells for this line seems reasonable to resolve.
The ones below in the deprecated endpoint don't need to since we'll be deleting it soon (hopefully) anyway